api/valor_api/backend/core/llm_clients.py

```diff
@@ -331,14 +403,17 @@ def _generate_bias_verdicts(
     def _coherence(
```

Comment:
RAGAS actually just added a new metric called coherence which does what the old coherence metric basically did. This is the description of the RAGAS coherence metric
"Does the submission present ideas, information, or arguments in a logical and organized manner?"

In the literature, coherence is usually this summarization specific version that I implemented. However I think it could be good to change the name to make the distinction more clear, now that there are two different versions of coherence out there. I could call this version SummaryCoherence, with name changes also to the relevant functions. 

I could also reimplement the non summary version of coherence to be similar to RAGAS's version, but that would be a follow up PR.

Reply:
using a more detailed name makes sense to me